admin/controllers/pupil.js

```diff
@@ -37,16 +42,16 @@ const getAddPupil = async (req, res, next) => {
 
 const postAddPupil = async (req, res, next) => {
   res.locals.pageTitle = 'Add pupil'
+  req.breadcrumbs(res.locals.pageTitle)
   let school
   try {
     school = await School.findOne({_id: req.body.school}).exec()
-    if (!school) {
-      throw new Error(`School [${req.body.school}] not found`)
-    }
+    // if (!school) {
```

Comment:
why not just delete it?

Reply:
Will be on the next commit, I forgot about it.